src/components/MovieHeader/MovieHeader.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useContext } from 'react';
+import styles from './MovieHeader.styl';
+import { useParams } from 'react-router';
+import MainContext from '../MainContext/MainContext';
+import Row from '../Row/Row';
+import Column from '../Column/Column';
+import Logo from '../Logo/Logo';
+import TextField from '../TextField/TextField';
+import { Link } from 'react-router-dom';
+import MovieDetails from '../MovieDetails/MovieDetails';
+import bgImg from '../../img/header-bg';
+
+const MovieHeader = () => {
+  const params = useParams();
+  const id = Number(params?.id);
```

Comment:
I'm just wondering, what if there is a broken id (some id that is not in `movieList`) it will go to ErrorBoundary, right?

Reply:
I was planned to implement redirect to 404 in this case